### PR TITLE
Task 619-624 Ohsumed abd Onestop_english datasets

### PR DESCRIPTION
**Pull request by papanisaicharan**

### CODE REVIEW

**Comment by swarooprm** [comment]
Reviews: https://github.com/swarooprm/CSE576HW/pull/14

**Comment by danyaljj** [comment]
Are these two datasets the same? 
```
    "Source": [
        "ohsumed dataset",
        "https://huggingface.co/datasets/viewer/?dataset=onestop_english"
    ],
```
if not, why have we included both of them?

**Comment by papanisaicharan** [comment]
> Are these two datasets the same?
> 
> ```
>     "Source": [
>         "ohsumed dataset",
>         "https://huggingface.co/datasets/viewer/?dataset=onestop_english"
>     ],
> ```
> 
> if not, why have we included both of them?

It was a mistake, I have corrected it now, please validate.

File: tasks/README.md [hunk 1]
```diff
@@ -504,6 +504,12 @@ Name | Summary | Category
 `task609_sbic_potentially_offense_binary_classification` | Determine whether the post is potentially offensive or not | Binary Classification
 `task610_conllpp_ner` | Recognize and label proper nouns (Named Entity Recognition) | NER/Labeling  
 `task615_moviesqa_answer_generation` | Given a question from an open movie database, generate an answer for that. | Answer Generation
+`task619_ohsumed_abstract_title_generation` | Generating title to Ohsumed dataset abstracts | Title Generation
+`task620_ohsumed_medical_subject_headings_answer_generation` | Generating MESH terms to Ohsumed dataset abstracts | Medical subject heading(MESH) term Generation
```
Comment (danyaljj) [inline tasks/README.md:610]: I haven't looked at this data yet, but in my experience, Mesh terms that are automatically fetched from indexing websites (e.g., NIH) are quite noisy. I would be in favor of including them, only if there is evidence on their quality (e.g., human evaluation).

Comment (papanisaicharan) [reply tasks/README.md:610]: @danyaljj, These will not be noisy as far as I know. 
All the MESH terms are handled properly [here](https://www.nlm.nih.gov/mesh/meshhome.html) and these MESH terms are assigned by humans after clear evaluation. So, there will not be any noisy data.
Please let me know if there are any issues.

Comment (danyaljj) [reply tasks/README.md:610]: Is there quantitative evidence on this?

**Comment by danyaljj** [comment]
Please make sure to set the `"Categories": [""]` and `"Domains": [""]` fields according to [the taxonomy presented here](https://github.com/allenai/natural-instructions-expansion/blob/master/doc/task-hierarchy.md).

**Comment by swarooprm** [comment]
@papanisaicharan, any update?

**Comment by papanisaicharan** [comment]
> @papanisaicharan, any update?

I really missed the email about the comments from the PR, I will update you ASAP.

**Comment by danyaljj** [comment]
- Dropping `task622_onestop_english_readability_classification` (classification into 3 classes) since the task is not quite clearly defined (and I am not sure how to improve it). 
 - Revised the wording of the remaining tasks.

**Pull request merged**